tests/test_charts.py

```diff
@@ -228,3 +232,13 @@ def test_tf_adjustment_chart(dialect, test_helpers):
 
     with pytest.raises(ValueError):
         linker.visualisations.tf_adjustment_chart("surname")
+
+@mark_with_dialects_excluding()
+def test_non_linker_charts(dialect):
+
+    url = "https://raw.githubusercontent.com/moj-analytical-services/splink_demos/master/demo_settings/real_time_settings.json"
```

Comment:
There are settings in Splink 4 format (with the dialect populated) here:
./docs/demos/demo_settings/saved_model_from_demo.json